content/citation-tags.tsv

```diff
@@ -277,3 +277,18 @@ Zintgraf2017_visualizing	arxiv:1702.04595
 goodfellow2016deep	url:http://www.deeplearningbook.org/
 li2016joint	url:http://www.ijcai.org/Proceedings/16/Papers/403.pdf
 world2004international	url:http://www.who.int/classifications/icd/en/
+ghahramani_protect	arxiv:1707.02476
+uncertainty_types	arxiv:1703.04977
+uncertainty_multi_task	arxiv:1705.07115
+guo_calibration	arxiv:1706.04599
```

Comment:
Can we cite the ICML version instead `url:http://proceedings.mlr.press/v70/guo17a.html`?